### DISCUSSION TITLE
This repository is confusing, is it for windows?

### DISCUSSION THREAD

**Discussion by Njasa2k** [ggml-org/llama.cpp#793] [Q&A] [answered]
from "The main goal is to run the model using 4-bit quantization on a MacBook" means I'm guessing it's for iOS and I have no business being in this repo.

There's a mix of windows and apple commands and I can't tell which. 
`# obtain the original LLaMA model weights and place them in ./models

ls ./models
65B 30B 13B 7B tokenizer_checklist.chk tokenizer.model`

Does the ls command even work on windows?

**Comment by x02Sylvie** [comment, marked as answer]
Instructions may be confusing but llama.cpp works both on windows, linux and mac

**Reply by Njasa2k** [reply]
How do I know which instructions is for windows and which instructions is for mac? is there a straight forward instruction?
I got 8GB of VRAM and 64GB of RAM, Windows, NVIDA RTX 2070.

**Reply by BetaDoggo** [reply]
The Windows equivalent of the `ls` command would be `dir`, but that command is just to check that the model files are in the correct location.

The rest of the instructions in that section should still apply but with `./quantize` replaced with `quantize.exe`, same with `./main` and `main.exe`.

**Reply by Wolvensdale** [reply]
Im trying to follow but "make" command won't work. Do you have some step by step for windows? I tried install cmake, and all visual studio, node js, etc etc all already installed.

**Reply by BetaDoggo** [reply]
I believe the build commands for cmake are the following:
```
cmake .
cmake --build . --config Release
```
Then I think the binaries will be put in the ./bin/Release folder.
